### PR TITLE
General YAML cleanup

### PR DESCRIPTION
**Pull request by aus**
Fixes a few problems I've personally had with parsing the YAML atomics:

- [x] Changed the string interpolation from `${foo}` to `#{foo}`
- [x] Fixed some input_arguments (may be more. I didnt validate everything)
- [x] Grepped and cleaned any non-ASCII characters
- [x] Removed “smart quotes”
- [x] Fix newlines at EOF

### CODE REVIEW

File: atomics/T1028/T1028.yaml [hunk 1]
```diff
@@ -35,7 +35,7 @@ atomic_tests:
   executor:
     name: command_prompt
     command: |
-      powershell.exe [activator]::CreateInstance([type]::GetTypeFromProgID("MMC20.application","${computername}")).Documnet.ActiveView.ExecuteShellCommand("c:\windows\system32\calc.exe", $null, $null, "7")
+      powershell.exe [activator]::CreateInstance([type]::GetTypeFromProgID("MMC20.application","#{computername}")).Documnet.ActiveView.ExecuteShellCommand("c:\windows\system32\calc.exe", $null, $null, "7")
```
Comment (brianebeyer) [inline atomics/T1028/T1028.yaml]: Looks like this should also be `computer_name` instead of `computername`

File: atomics/T1033/T1033.yaml [hunk 2]
```diff
@@ -20,11 +20,11 @@ atomic_tests:
     command: |
       cmd.exe /C whoami
       wmic useraccount get /ALL
-      quser /SERVER:"${computername}"
+      quser /SERVER:"#{computername}"
```
Comment (brianebeyer) [inline atomics/T1033/T1033.yaml]: `computer_name`

File: atomics/T1033/T1033.yaml [hunk 3]
```diff
@@ -20,11 +20,11 @@ atomic_tests:
...
       quser
-      qwinsta.exe" /server:${computername}
+      qwinsta.exe" /server:#{computername}
```
Comment (brianebeyer) [inline atomics/T1033/T1033.yaml]: `computer_name`

File: atomics/T1033/T1033.yaml [hunk 4]
```diff
@@ -20,11 +20,11 @@ atomic_tests:
...
       qwinsta.exe
-      for /F “tokens=1,2” %i in (‘qwinsta /server:${computername} ^| findstr “Active Disc”‘) do @echo %i | find /v “#” | find /v “console” || echo %j > usernames.txt
+      for /F “tokens=1,2” %i in (‘qwinsta /server:#{computername} ^| findstr “Active Disc”‘) do @echo %i | find /v “#” | find /v “console” || echo %j > usernames.txt
```
Comment (brianebeyer) [inline atomics/T1033/T1033.yaml]: `computer_name`

Also, are those what I call "evil quotes" that are the funky non-ASCII quotes?

**Comment by aus** [comment]
Thanks for catching those @brianebeyer. I added a few additional commits to cleanup some other problems too. And I definitely like the idea of fleshing out the validation script more for validating input arguments.

Ready for review.

**Comment by brianebeyer** [comment]
Thanks @aus! Are you interested in taking a stab at updating the validation script?

**Comment by aus** [comment]
I'll try and take a look next week. I have a few ideas to implement. Thanks for the merge!

**Pull request merged**